foundations/installations/setting_up_git.md

```diff
@@ -128,6 +128,13 @@ git config --get user.name
 git config --get user.email
 ~~~
 
+<strong>For macOS users only</strong> - run these two commands to get GitHub to ignore .DS_Store files, which are automatically created when you use Finder to look into a folder. .DS_Store files are invisible to the user and hold custom attributes or metadata (like thumbnails) for the folder, and if you don't configure GitHub to ignore them, pesky .DS_Store files will show up in your commits.
```

Comment:
Looks good, we generally use markdown formatting in the lesson content though, so let's swap that `<strong>` tag for `**`s instead. Notes are normally just direct as well.

Good work, once these changes are made we can merge. :D

```suggestion
**macOS Users:** Run these two commands to tell git to ignore .DS_Store files, which are automatically created when you use Finder to look into a folder. .DS_Store files are invisible to the user and hold custom attributes or metadata (like thumbnails) for the folder, and if you don't configure GitHub to ignore them, pesky .DS_Store files will show up in your commits.
```